src/helpers/todosFiltering.ts

```diff
@@ -0,0 +1,35 @@
+import { Todo } from '../types/Todo';
+import { FilteringOptions } from '../types/FilteringOptions';
+
+export const getTodosFilteredByStatus = (
+  todos: Todo[],
+  filteringOption: FilteringOptions,
+) => {
+  return todos.filter(({ completed }) => {
+    switch (filteringOption) {
+      case FilteringOptions.Active:
+        return !completed;
+
+      case FilteringOptions.Completed:
+        return completed;
+
+      default:
+        break;
```

Comment:
Replace 'break' with 'return true' to include all todos in the case of FilteringOptions.All